edx_event_bus_kafka/config.py

```diff
@@ -16,10 +19,13 @@
 
 
 # return type (Optional[SchemaRegistryClient]) removed from signature to avoid error on import
-def create_schema_registry_client():
+@lru_cache
```

Comment:
Ahhh, good call. I forgot that one existed!

Unbounded caching makes me nervous (memory leaks), and I'd usually prefer a large maxsize over no maxsize, but it's fine here.

Reply:
1. This is bounded, right? Not bounded by cache size, but bounded by max of single cached value.
2. I think your comment means you plan to update this, and I'm just sweeping in before this is ready for review?

Reply:
Adding to my last comment, now that I noticed you may have switched to `@cache` and then back to `@lru_cache`.
1. I still prefer `@cache`.
2. I think `@lru_cache(maxsize=1)` would be less misleading, because we're not really using the LRU part of the cache.
3. Although I found it distracting, I don't think it is very important, so feel free to leave as-is if you have some reason why you still think it is better. :)

Reply:
Oh yeah, I mentioned it in the commit message but not in a comment -- `functools.cache` is new in Python 3.9.

Reply:
It's definitely bounded by being a nullary function. I guess we could set `maxsize=1` -- there's a very small risk that more args are added in the future, but maybe not something worth worrying about? Or I could add a comment like `@lru_cache  # expecting just a single cache entry`.

Reply:
I think I like `@lru_cache # expecting just a single cache entry` the best, but whatever you decide is fine.